gallery/visualtests/testUtils.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+const { Target } = require('@applitools/eyes-webdriverio');
+
+module.exports = {
+  simpleTest(selector) {
+    return async () => {
```

Comment:
Can we use `async` and `await` now? 😮

Reply:
> Can we use async and await now? open_mouth

I'm pretty sure we've been able to for some time.  That said, these files are run directly in node, not put through webpack or anything, so the ability to use async/await here doesn't imply anything about whether it is supported in application code.

I hadn't found async/await particularly necessary in our application code, as we usually have fairly succinct asychronous chains in any given method.  In these tests though, virtually every operation is async so it really helps. (Also, the example code that I started from was written in this style which is really what prompted me to use it 🙂).